includes/admin/class-amp-post-meta-box.php

```diff
@@ -202,7 +202,9 @@ public function enqueue_block_assets() {
 			'errorMessages' => $error_messages,
 		);
 
-		$script_data['i18n'] = function_exists( 'wp_get_jed_locale_data' ) ? wp_get_jed_locale_data( 'amp' ) : gutenberg_get_jed_locale_data( 'amp' );
+		if ( function_exists( 'wp_set_script_translations' ) ) {
```

Comment:
Should this still support older versions as well where the `wp_set_script_translations` was not introduce yet?

Reply:
I guess the assumption is that Gutenberg will be included in 5.0 and not before, so it would be okay like this as well.

Reply:
Hi @miina,
That's a really good question . This change would leave out people that use some version of Gutenberg, but WordPress earlier than [5-0-beta-4](https://wordpress.org/news/2018/11/wordpress-5-0-beta-4/) (when `wp_set_script_translations()` was released). 

So they wouldn't get translations for the meta box.

Should we expect users that are running Gutenberg to have at least WordPress [5-0-beta-4](https://wordpress.org/news/2018/11/wordpress-5-0-beta-4/)?

I think it'll add a lot of complexity to have conditionals to support this.

```
if ( function_exists( 'wp_set_script_translations' ) ) {
	wp_set_script_translations( self::BLOCK_ASSET_HANDLE, 'amp' );
} elseif ( function_exists( 'wp_get_jed_locale_data' ) ) {
	$script_data['i18n'] = wp_get_jed_locale_data( 'amp' );
} elseif ( function_exists( 'wp_get_jed_locale_data' ) ) {
	$script_data['i18n'] = gutenberg_get_jed_locale_data( 'amp' );
}
```

 If we're going to remove references to `wp_set_script_translations()` and `wp_get_jed_locale_data()`, we should do it before the `1.0` stable release.

Reply:
Still, you raise a good point that this should support older versions. I'm going to add conditionals that support WP before `5.0-beta4`.

Reply:
https://github.com/Automattic/amp-wp/pull/1634/commits/0cac87cc5ed3c310b973dc8e5acdbb415460cadd restores the previous use of `wp_get_jed_locale_data()` or `gutenberg_get_jed_locale_data()` `wp_set_script_translations()` doesn't exist. I still need to address the failed [Travis build](https://travis-ci.org/Automattic/amp-wp/jobs/458111904).

Reply:
We don't need to sorry about back compat for WordPress 5.0 beta releases. Only be concerned with 4.9.x+Gutenberg and 5.0 stable.